assets/js/session/index.js

```diff
@@ -427,7 +427,9 @@ function handleDocumentMouseDown(hook, event) {
   }
 
   // Find the focusable element, if one was clicked
-  const focusableEl = event.target.closest(`[data-focusable-id]`);
+  const focusableEl = event.target.closest
+    ? event.target.closest(`[data-focusable-id]`)
+    : false;
```

Comment:
So I'd leave it as is then :)

```suggestion
  const focusableEl = event.target.closest(`[data-focusable-id]`);
```